README.md

```diff
@@ -13,6 +13,6 @@ Visit [https://whereby.dev](https://whereby.dev) to view the full Embedded devel
 
 Whereby publishes these examples to help the developer community understand how the Whereby Embedded product can be implemented.
 
-Whereby does not recommend using such examples in a production environment without a prior assessment and appropriate testing relevant to the production setup targeted which can be of operational, technical, security or legal (e.g. library licenses assessment) nature.
+Whereby does not recommend using such examples in a production environment without a prior assessment and appropriate testing relevant to the production setup targeted which can be of operational, technical, security or legal (e.g. library licenses assessment) nature. You expressly agree that the use of these examples is at your sole risk.
 
-These examples do not create any warranties, representations, contractual commitments, conditions or assurances from Whereby, its affiliates, suppliers or licensors.
+Whereby, its affiliates, suppliers, or licensors, whether express or implied, do not make any representation, warranties, contractual commitments, conditions, or assurances by the operation of these examples, or the information, content, materials (including …you can add example if you wish), therein.
```

Comment:
`…you can add example if you wish` we should add something in there or just remove the whole `()` right?